css-var-polyfill.js

```diff
@@ -63,7 +63,7 @@ let cssVarPoly = {
   // find all the "--variable: value" matches in a provided block of CSS and add them to the master list
   findSetters: function(theCSS, counter) {
     // console.log(theCSS);
-    cssVarPoly.varsByBlock[counter] = theCSS.match(/(--.+:.+;)/g) || [];
+    cssVarPoly.varsByBlock[counter] = theCSS.match(/(--[^:)]+:[\s]*[^;}]+)/g) || [];
```

Comment:
This doesn't pull in the `;` after the declaration. Did you find that it wasn't needed? it looks like it could be added by adding a `;` after the `+`.  It has been forever since I made this in the first place, and haven't really used it since, that I don't know what is needed or not =P

`(--[^:)]+:[\s]*[^;}]+;)`

Reply:
Yeah it's left out on purpose. CSS does not require a `;` if it is the last item in the class so this is usually removed on a minified file.

Reply:
See test here: https://regex101.com/r/lBSZQZ/3
What might be even better is `(--[^:)]+:[\s]*[^;}\n]+)` so it stops at a newline also.

Reply:
Thanks!